src/brasil/gov/agenda/browser/templates/compromissoview.pt

```diff
@@ -2,15 +2,16 @@
     xmlns:metal="http://xml.zope.org/namespaces/metal"
     xmlns:tal="http://xml.zope.org/namespaces/tal"
     xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-    metal:use-macro="context/main_template/macros/master">
+    metal:use-macro="context/main_template/macros/master"
+    i18n:domain="brasil.gov.agenda">
 
     <metal:header fill-slot="header">
         <dl class="portalMessage info"
             tal:condition="view/editable">
             <dt i18n:translate="">Info</dt>
             <dd i18n:translate="">
-              Você vê esta página porque possui permissão para editá-la. Outros usuários serão redirecionados para a
-              visão da Agenda Diária
+              Voce ve esta pagina porque possui permissao para edita-la. Outros usuarios serao redirecionados para a
```

Comment:
i18ndude chokes when using *.pt files with accents. Since there's pt_BR/LC_MESSAGES/brasil.gov.agenda.po, the accented words were put in there. But we do realize we should have said that in the commit message to avoid confusion.